Gems/Atom/RHI/Code/Include/Atom/RHI/SwapChain.h

```diff
@@ -81,6 +81,13 @@ namespace AZ
 
             AZ_RTTI(SwapChain, "{888B64A5-D956-406F-9C33-CF6A54FC41B0}", Object);
 
+#if defined(PAL_TRAIT_LINUX_WINDOW_MANAGER_XCB)
+            // On Linux platforms that uses XCB, a resize may occur in the swap chain but the command queue may still
```

Comment:
Please add a GHI issue using the following format so this hack doesn't get lost and easily tracked
//[GFX TODO][GHI - XXXX] - On Linux platforms that....